hiplot_mlflow.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2020 Faculty Science Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+from typing import Union
+from urllib.parse import parse_qs, urlparse
+
+import hiplot
+from mlflow import search_runs, tracking
+from numpy import finfo, float64
+from pandas import DataFrame
+
+_positive_inf = finfo(float64).max
+_negative_inf = finfo(float64).min
+
+
+def fetch(name: str, include_tags: bool = False) -> hiplot.Experiment:
+    """Fetch experiments from the MLflow experiments tracking,
+    by experiment name.
+
+    Parameters
+    ----------
+    name: str
+        The name of the tracked experiment to look up
+    include_tags: bool
+        Whether to include tags as well in the resulting HiPlot experiment (False)
+
+    Returns
+    -------
+    hiplot.Experiment
+        The resulting experiment
+    """
+    return fetch_by_id(_get_experiment_id_from_name(name), include_tags)
+
+
+def fetch_by_id(id: Union[int, str], include_tags: bool = False) -> hiplot.Experiment:
+    """Fetch experiments from the MLflow experiments tracking,
+    by experiment ID.
+
+    Parameters
+    ----------
+    id: int|str
+        The ID if the experiment to look up
+    include_tags: bool
+        Whether to include tags as well in the resulting HiPlot experiment (False)
+
+    Returns
+    -------
+    hiplot.Experiment
+        The resulting experiment
+    """
+    df = search_runs([id])
+    return _create_experiment_from_dataframe(df, include_tags)
+
+
+def fetch_by_uri(uri: str) -> hiplot.Experiment:
+    """Fetch experiments from the MLflow experiments tracking,
+    by full URI.
+
+    The URI can take these forms:
+    * `mlflow://id/<id>` to look up experiments by ID
```

Comment:
`numpydoc` docstrings are intepreted as reStructuredText, so if you want these to result in fixed-width formatting when rendered, you need to use double backticks, i.e.:
```suggestion
    * ``mlflow://id/<id>`` to look up experiments by ID
```